charmcraft/extensions/gunicorn.py

```diff
@@ -0,0 +1,245 @@
+            },
+            "create-superuser": {
+                "description": "Create a new Django superuser account.",
+                "params": {"username": {"type": "string"}, "email": {"type": "string"}},
+                "required": ["username", "email"],
+            },
+        },
+    }
+
+    _OPTIONS = {
+        "flask": {
+            "flask-application-root": {
+                "type": "string",
+                "description": "Path in which the application / web server is mounted. This configuration will set the FLASK_APPLICATION_ROOT environment variable. Run `app.config.from_prefixed_env()` in your Flask application in order to receive this configuration.",
+            },
+            "flask-debug": {
+                "type": "boolean",
+                "description": "Whether Flask debug mode is enabled.",
+            },
+            "flask-env": {
+                "type": "string",
+                "description": "What environment the Flask app is running in, by default it's 'production'.",
+            },
+            "flask-permanent-session-lifetime": {
+                "type": "int",
+                "description": "Time in seconds for the cookie to expire in the Flask application permanent sessions. This configuration will set the FLASK_PERMANENT_SESSION_LIFETIME environment variable. Run `app.config.from_prefixed_env()` in your Flask application in order to receive this configuration.",
+            },
+            "flask-preferred-url-scheme": {
+                "type": "string",
+                "default": "HTTPS",
+                "description": 'Scheme for generating external URLs when not in a request context in the Flask application. By default, it\'s "HTTPS". This configuration will set the FLASK_PREFERRED_URL_SCHEME environment variable. Run `app.config.from_prefixed_env()` in your Flask application in order to receive this configuration.',
+            },
+            "flask-secret-key": {
+                "type": "string",
+                "description": "The secret key used for securely signing the session cookie and for any other security related needs by your Flask application. This configuration will set the FLASK_SECRET_KEY environment variable. Run `app.config.from_prefixed_env()` in your Flask application in order to receive this configuration.",
+            },
+            "flask-session-cookie-secure": {
+                "type": "boolean",
+                "description": "Set the secure attribute in the Flask application cookies. This configuration will set the FLASK_SESSION_COOKIE_SECURE environment variable. Run `app.config.from_prefixed_env()` in your Flask application in order to receive this configuration.",
+            },
+        },
+        "django": {
+            "django-debug": {
+                "type": "boolean",
+                "description": "Whether Django debug mode is enabled.",
```

Comment:
Should we be explicit about the default for this?

Reply:
Yes, indeed, updated, thanks!